src/assets/custom/css/_sass/_in-page-banner.scss

```diff
@@ -3,6 +3,8 @@ $banner-height: 300px;
 .in-page-banner {
   min-height: $banner-height;
   overflow: hidden;
+  max-width: 1300px;
```

Comment:
```suggestion
  max-width: $extraLargeMin;
```

Reply:
Could possibly use this variable - but I guess this will need adjusting again when the Homepage get's fully reworked

Reply:
sounds good - let's save this for a future iteration